app/services/open_library_service.rb

```diff
@@ -0,0 +1,17 @@
+class OpenLibraryService
+  def book_info(isbn)
+    url = 'https://openlibrary.org/api/books?bibkeys=ISBN:' + isbn + '&format=json&jscmd=data'
```

Comment:
A few things about this:
- Usually base URLs (in this case `'https://openlibrary.org/api/'`) should be either in constants in the class (`BASE_URL = 'https://openlibrary.org/api/'`) or environment variables in secrets. If you use HTTParty classes, use the `base_uri` helper.
- For the second part, string interpolation is usually cleaner. `"books?bibkeys=ISBN:#{isbn}&format=json&jscmd=data"`

I'd really recommend using an HTTParty class instead. See [the documentation](https://github.com/jnunemaker/httparty) in the example with `base_uri`. The way you use it won't change much